src/map/modifier.h

```diff
@@ -800,8 +803,7 @@ enum class Mod
 
     // The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
     // 570 through 825 used by WS DMG mods these are not spares.
-    // SPARE = 964, // stuff
-    // SPARE = 965, // stuff
+    // SPARE = 967, // stuff
```

Comment:
I went and checked the other PRs for what the highest IDs should be
```
    // SPARE = 967, // stuff
    // SPARE = 968, // stuff
    // SPARE = 969, // stuff
```
and status.lua should match:
```lua
    -- SPARE = 967, -- stuff
    -- SPARE = 968, -- stuff
    -- SPARE = 969, -- stuff
```

Reply:
Just to be clear, you're saying I just need to update which are the new current spare IDs not change which IDs I'm using?

Reply:
yeah its a matter of keeping them up to date and in sync so that people don't accidentally use twice. we have 4 broken mods right now from it that need fixed. (issue 998)

the next current free mods ID is 972 (I have edited my review comment to note this)

its a pain point for contributors when multiple prs add a mod at the same time, because whoever gets merged first the other has to update their pr and there is no avoiding it.

Reply:
Thanks for the clarification, as requested, I've updated them